2021-05-21 22:11:36,127 INFO    MainThread:274330 [wandb_setup.py:_flush():69] setting env: {}
2021-05-21 22:11:36,127 INFO    MainThread:274330 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-21 22:11:36,127 INFO    MainThread:274330 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Mask-Detection-PT-V2/wandb/run-20210521_221136-3mcobpn8/logs/debug.log
2021-05-21 22:11:36,127 INFO    MainThread:274330 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Mask-Detection-PT-V2/wandb/run-20210521_221136-3mcobpn8/logs/debug-internal.log
2021-05-21 22:11:36,134 INFO    MainThread:274330 [wandb_init.py:init():369] calling init triggers
2021-05-21 22:11:36,134 INFO    MainThread:274330 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-21 22:11:36,134 INFO    MainThread:274330 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 6lx3zxei
2021-05-21 22:11:36,135 INFO    MainThread:274330 [wandb_run.py:finish():1208] finishing run ranuga-d/Face-Mask-Detection/6lx3zxei
2021-05-21 22:11:36,138 INFO    MainThread:274330 [jupyter.py:save_history():349] saving 19 cells to _session_history.ipynb
2021-05-21 22:11:36,142 INFO    MainThread:274330 [wandb_run.py:_config_callback():787] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2021-05-21 22:11:36,154 INFO    MainThread:274330 [jupyter.py:save_ipynb():268] looking for notebook: train.ipynb
2021-05-21 22:11:36,676 INFO    MainThread:274330 [wandb_init.py:_jupyter_teardown():267] saved code and history
2021-05-21 22:11:36,676 INFO    MainThread:274330 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-21 22:11:36,676 INFO    MainThread:274330 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-21 22:11:36,676 INFO    MainThread:274330 [wandb_run.py:_restore():1507] restore
2021-05-21 22:11:41,680 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-21 22:11:42,666 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 18477
  total_bytes: 18477
  deduped_bytes: 11203
}

2021-05-21 22:11:43,222 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 18477
  total_bytes: 18477
  deduped_bytes: 11203
}

2021-05-21 22:11:43,919 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18477
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,023 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 35037
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,126 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 35119
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,228 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,331 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,433 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,534 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,636 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,738 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,839 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:44,941 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:45,043 INFO    MainThread:274330 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 42029
  total_bytes: 42029
  deduped_bytes: 11203
}

2021-05-21 22:11:46,421 INFO    MainThread:274330 [wandb_run.py:_show_files():1879] logging synced files
2021-05-21 22:11:46,426 INFO    MainThread:274330 [wandb_init.py:init():418] starting backend
2021-05-21 22:11:46,426 INFO    MainThread:274330 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-21 22:11:46,430 INFO    MainThread:274330 [backend.py:ensure_launched():132] starting backend process...
2021-05-21 22:11:46,469 INFO    MainThread:274330 [backend.py:ensure_launched():137] started backend process with pid: 276210
2021-05-21 22:11:46,470 INFO    MainThread:274330 [wandb_init.py:init():423] backend started and connected
2021-05-21 22:11:46,482 INFO    MainThread:274330 [wandb_init.py:init():465] updated telemetry
2021-05-21 22:11:46,482 INFO    MainThread:274330 [wandb_init.py:init():484] communicating current version
2021-05-21 22:11:47,588 INFO    MainThread:274330 [wandb_init.py:init():489] got version response 
2021-05-21 22:11:47,588 INFO    MainThread:274330 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-21 22:11:48,108 INFO    MainThread:274330 [wandb_init.py:init():522] starting run threads in backend
2021-05-21 22:11:53,114 INFO    MainThread:274330 [wandb_run.py:_console_start():1565] atexit reg
2021-05-21 22:11:53,115 INFO    MainThread:274330 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-21 22:11:53,115 INFO    MainThread:274330 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-21 22:11:53,115 INFO    MainThread:274330 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-21 22:11:53,115 INFO    MainThread:274330 [wandb_init.py:init():546] run started, returning control to user process
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_setup.py:_flush():69] setting env: {}
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Mask-Detection-PT-V2/wandb/run-20210521_221249-26nqdb6g/logs/debug.log
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/Mask-Detection-PT-V2/wandb/run-20210521_221249-26nqdb6g/logs/debug-internal.log
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8b612dd7f0>
2021-05-21 22:12:49,272 INFO    MainThread:274330 [wandb_init.py:init():369] calling init triggers
2021-05-21 22:12:49,273 INFO    MainThread:274330 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-21 22:12:49,273 INFO    MainThread:274330 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 3mcobpn8
2021-05-21 22:12:49,273 INFO    MainThread:274330 [wandb_run.py:finish():1208] finishing run ranuga-d/Face-Mask-Detection/3mcobpn8
